presto-main/src/main/java/io/prestosql/sql/planner/LocalDynamicFilter.java

```diff
@@ -173,9 +198,14 @@ private static boolean isFilterAboveTableScan(PlanNode node)
         return buildChannels;
     }
 
-    public ListenableFuture<Map<Symbol, Domain>> getResultFuture()
+    public ListenableFuture<Map<String, Domain>> getDynamicFilterDomains()
+    {
+        return Futures.transform(resultFuture, this::convertTupleDomain, directExecutor());
+    }
+
+    public ListenableFuture<Map<Symbol, Domain>> getNodeLocalDynamicFilterForSymbols()
```

Comment:
nit: Local specific logic should be moved out of `LocalDynamicFilter` eventually